canonical-web-and-design/charmhub.io
# Build charm resources page

## Done
Built the resources section of the charm details page

## QA
- Go to https://charmhub-io-1082.demos.haus/mlmd/resources/oci-image
- Go to https://charmhub-io-1082.demos.haus/postgresql/resources/wal-e
- Check that the pages look OK

[Designs](https://app.zeplin.io/project/60abb99a154f9448324fcbfe/screen/60dc831ea1e31c05f2bc33fb)

## Issue
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/2134
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/2130

## Screenshots

![resources_oci](https://user-images.githubusercontent.com/501889/125755524-bc043863-46c5-492e-8e56-6651561426ce.png)

![resources_file](https://user-images.githubusercontent.com/501889/125755546-2fa12aa3-f41e-4e97-9518-6375dc3b3bcd.png)
